open_instruct/grpo_fast.py

```diff
@@ -1826,9 +1826,25 @@ def sync_weights_and_prepare_prompts(
     return queries_next, ground_truths_next, datasets_next, dataset_indices
 
 
-def load_data_from_packing_thread(packed_sequences_Q: Queue, num_total_tokens: int):
+def load_data_from_packing_thread(
+    packed_sequences_Q: Queue, num_total_tokens: int, stop_generate_event: threading.Event
```

Comment:
If we're passing stop_generate_event through to multiple threads, let's just call it `stop_event`?

Reply:
done!